src/Verifier/SAW/Simulator/BitBlast.hs

```diff
@@ -241,6 +241,11 @@ prims be =
   , Prims.bpIntLt  = pure2 (\x y -> AIG.constant be (x < y))
   , Prims.bpIntMin = pure2 min
   , Prims.bpIntMax = pure2 max
+
+    -- Array operations
+  , Prims.bpArrayConstant = error "bpArrayConstant unimplemented for backend"
+  , Prims.bpArrayLookup = error "bpArrayLookup unimplemented for backend"
+  , Prims.bpArrayUpdate = error "bpArrayUpdate unimplemented for backend"
```

Comment:
Ditto my comments on `saw-core` and `saw-core-sbv`.

Reply:
Same as what I said about `saw-core-sbv`: We should avoid using `error` if at all possible. The evaluation monad here is `IO`, so some better error-handling options are available.